pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -89,6 +89,8 @@ const (
 	AccountReady AccountConditionType = "Ready"
 	// AccountFailed is set when account creation has failed
 	AccountFailed AccountConditionType = "Failed"
+	// AccountCreationFailed is set when an error occured during account creations
```

Comment:
Would you mind making this "...during AWS account creation" to specify that it was a "real" account creation that failed, and not our operator CR object that failed?  Just to be clear what the difference is until the "AccountFailed" condition goes away.